README.md

```diff
@@ -42,8 +44,19 @@ sudo SNIFFER_CONFIG=./configuration/SnifferConfigurationFile.json ./sniffer
         }
     ]
 }
+```
+## Troubleshooting
 
+```json
+{"level":"fatal", "msg":"error starting the container watcher", "error":"failed to initialize container collection: starting runc fanotify: no runc instance can be monitored with fanotify. The following paths were tested: /bin/runc,/usr/bin/runc,/usr/sbin/runc,/usr/local/bin/runc,/usr/local/sbin/runc,/usr/lib/cri-o-runc/sbin/runc,/run/torcx/unpack/docker/bin/runc,/usr/bin/crun. You can use the RUNC_PATH env variable to specify a custom path. If you are succesful doing so, please open a PR to add your custom path to runcPaths\n"}
 ```
+
+In case you read this error from the `node-agent` logs, you can solve by re-install the **helm-chart** with the following parameters, replacing the `</path/to/your/runc>` with your own path:
```

Comment:
should we cross-reference this (or move) into the Helm chart README?

Reply:
Good question.. maybe is better cross-referencing it. Don't have a strong opinion on that.

Reply:
I changed my mind, I think this needs to be mentioned under the helm-chart repository.
This error is related only to the installation with helm itself.